bank-service/src/test/java/rs/banka4/bank_service/integration/order/OrderServiceTest.java

```diff
@@ -33,6 +33,7 @@
 import rs.banka4.testlib.integration.DbEnabledTest;
 
 @SpringBootTest
+@ActiveProfiles("test")
```

Comment:
why is this necessary?  the test profile should be enabled by `bank-service/src/test/resources/application.yml` already